pkg/api/handlers/version.go

```diff
@@ -30,7 +35,53 @@ func (v *VersionHandler) getVersionResponse() map[string]string {
 			values["error"] = err.Error()
 		}
 	}
-	values["otto"] = version.Get().String()
+	values["acorn"] = version.Get().String()
+	values["gptscript"] = v.gptscriptVersion
 	values["emailDomain"] = v.emailDomain
 	return values
 }
+
+const gptscriptModulePath = "github.com/gptscript-ai/gptscript"
+
+func getGPTScriptVersion() string {
+	bi, _ := debug.ReadBuildInfo()
+
+	var gptscriptVersion string
+	for _, dep := range bi.Deps {
+		if dep.Path == gptscriptModulePath {
+			gptscriptVersion = simplifyModuleVersion(dep.Version)
+			break
+		}
+	}
+
+	return gptscriptVersion
+}
+
+// simplifyModuleVersion returns a simplified variant of a given module version string.
+// If the given version is a Go pseudo-version, it strips the timestamp and truncates the revision to the first 7 characters.
+// Empty strings and non-Go pseudo-versions are returned unaltered.
+func simplifyModuleVersion(version string) string {
+	if version == "" || !module.IsPseudoVersion(version) {
+		return version
+	}
+
+	// Extract the base version (tag) and revision (commit hash)
+	// Panic on failure. This should never happen for well-formed pseudo-versions.
+	base, err := module.PseudoVersionBase(version)
+	if err != nil {
+		panic(fmt.Errorf("error extracting base version: %w", err))
```

Comment:
why are we panicing? would be ideal if version api fails we can make the whole server crash

Reply:
this happens once on startup and would only panic with a bad module version, which IIUC, shouldn't be possible at runtime. If we're scared of the panic statement, I can remove it.

Reply:
I removed the panic statements to reduce reviewer stress 🙂

Reply:
haha ok. Make sense. 👍